cfme/tests/infrastructure/test_vm_rest.py

```diff
@@ -21,12 +22,25 @@ def vm_name(request, a_provider, rest_api):
 
 
 @pytest.mark.tier(3)
-@pytest.mark.parametrize("from_detail", [True, False], ids=["from_detail", "from_collection"])
-def test_delete(vm_name, rest_api, from_detail):
+@pytest.mark.parametrize('method', ['post', 'delete'])
```

Comment:
**Optional**
Consider adding ids for post and delete and capitalize them. I think this will make it more obvious in the test output that its the HTTP method.